h2o-core/src/main/java/water/H2O.java

```diff
@@ -529,7 +524,6 @@ else if (s.matches("ga_hadoop_ver")) {
       else if (s.matches("ga_opt_out")) {
```

Comment:
Do you want to remove this path?

Reply:
I didn't - because that would break peoples scripts.